nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/test/java/org/apache/nifi/processors/iceberg/TestIcebergRecordConverter.java

```diff
@@ -473,6 +516,103 @@ public void testPrimitives(FileFormat format) throws IOException {
         }
     }
 
+    @DisabledOnOs(WINDOWS)
+    @ParameterizedTest
+    @EnumSource(value = FileFormat.class, names = {"AVRO", "ORC", "PARQUET"})
```

Comment:
One general note on these tests, it it probably worth avoiding some of the repeated tests in the interest of execution time. Several other tests ran through all formats, requiring over a minute to exercise all cases. Keeping unit test method runs under a second would be ideal, so these new tests may need to be more narrowly scoped to avoid excessive running time.